routes/movies.routes.js

```diff
@@ -0,0 +1,69 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+// all your routes here
+const Celebrity = require("../models/Celebrity.model");
+
+const Movie = require("../models/Movie.model");
+
+router.get("/movies", async (req, res, next) => {
+
+  try {
+    let movies = await Movie.find();
+
+    res.render("celebrities/movies", { movies });
+  } catch (error) {
+    console.log(error);
+    next(error);
+  }
+});
+
+router.get("/movies/create", (req, res, next) => {
+    const celebrities = Celebrity.find()
+    res.render("celebrities/new-movies", { celebrities });
+  });
+  
+  router.post("/movies/create", async (req, res, next) => {
+
+try {
+        const { title, genre, plot, cast  } = req.body;
+    
+        await Movie.create({ title, genre, plot, cast });
+    
+        res.redirect("/movies");
+      } catch (error) {
+        console.log(error);
+        next(error);
+      }
+    });
+
+router.get("/movies", async (req, res, next) => {
```

Comment:
On line 9, you have the same `'/movies'` route and that's creating a problem. Your routes have to be unique!